packages/ember-htmlbars/tests/helpers/get_test.js

```diff
@@ -1,495 +0,0 @@
-  });
-
-  runAppend(view);
-
-  equal(view.$().text(), '[red] [red]', 'should return \'red\'');
-
-  run(function() {
-    view.set('context.objectKey', 'colors2');
-  });
-
-  equal(view.$().text(), '[green] [green]', 'should return \'green\'');
-
-  run(function() {
-    view.set('context.objectKey', 'colors1');
-  });
-
-  equal(view.$().text(), '[red] [red]', 'should return \'red\'');
-
-  run(function() {
-    view.set('context.key', 'key2');
-  });
-
-  equal(view.$().text(), '[yellow] [yellow]', 'should return \'yellow\'');
-
-  run(function() {
-    view.set('context.objectKey', 'colors2');
-  });
-
-  equal(view.$().text(), '[purple] [purple]', 'should return \'purple\'');
-
-  run(function() {
-    view.set('context.objectKey', 'colors1');
-  });
-
-  equal(view.$().text(), '[yellow] [yellow]', 'should return \'yellow\'');
-});
-
-QUnit.test('should handle object values as nulls', function() {
-  var context = {
-    colors: null
-  };
-
-  view = EmberView.create({
-    context: context,
-    template: compile('[{{get colors \'apple\'}}] [{{if true (get colors \'apple\')}}]')
-  });
-
-  runAppend(view);
-
-  equal(view.$().text(), '[] []', 'should return \'\' for {{get colors \'apple\'}} (colors = null)');
-
-  run(function() {
-    view.set('context.colors', { apple: 'green', banana: 'purple' });
-  });
-
-  equal(view.$().text(), '[green] [green]', 'should return \'green\' for {{get colors \'apple\'}} (colors = { apple: \'green\', banana: \'purple\' })');
-
-  run(function() {
-    view.set('context.colors', null);
-  });
-
-  equal(view.$().text(), '[] []', 'should return \'\' for {{get colors \'apple\'}} (colors = null)');
-});
-
-QUnit.test('should handle object keys as nulls', function() {
-  var context = {
-    colors: { apple: 'red', banana: 'yellow' },
-    key: null
-  };
-
-  view = EmberView.create({
-    context: context,
-    template: compile('[{{get colors key}}] [{{if true (get colors key)}}]')
-  });
-
-  runAppend(view);
-
-  equal(view.$().text(), '[] []', 'should return \'\' for {{get colors key}}  (key = null)');
-
-  run(function() {
-    view.set('context.key', 'banana');
-  });
-
-  equal(view.$().text(), '[yellow] [yellow]', 'should return \'yellow\' for {{get colors key}} (key = \'banana\')');
-
-  run(function() {
-    view.set('context.key', null);
-  });
-
-  equal(view.$().text(), '[] []', 'should return \'\' for {{get colors key}}  (key = null)');
-});
-
-QUnit.test('should handle object values and keys as nulls', function() {
-  var context = {
-    colors: null,
-    key: null
-  };
-
-  view = EmberView.create({
-    context: context,
-    template: compile('[{{get colors key}}] [{{if true (get colors key)}}]')
-  });
-
-  runAppend(view);
-
-  equal(view.$().text(), '[] []', 'should return \'\' for {{get colors key}}  (colors=null, key = null)');
-});
-
-QUnit.test('get helper value should be updatable using {{input}} and (mut) - dynamic key', function() {
```

Comment:
:heavy_check_mark: get helper value should be updatable using {{input}} and (mut) - dynamic key